packages/components/molecules/f-breadcrumbs/test/specs/accessibility/axe-accessibility.spec.js

```diff
@@ -0,0 +1,8 @@
+import { getAccessibilityTestResults } from '../../../../../../../test/utils/axe-helper';
+
+describe('Accessibility tests', () => {
+    it('a11y - should test f-breadCrumbs component WCAG compliance', () => {
+        // Act
+        const axeResults = getAccessibilityTestResults('f-breadCrumbs');
```

Comment:
Just to check  – should this be `f-breadcrumbs` to match the `package.json` name, or is the camelCase ok?

Reply:
I am not sure tbh, @siggerzz is probably best to answer that question? if it needs changing can we also change in the generator.

Reply:
Yeah, I'd rename this to f-breadcrumbs.

Reply:
done ditto below

Reply:
Should this be `f-breadcrumbs` now?